pkg/catalog/catalog.go

```diff
@@ -1768,6 +1768,12 @@ type UncommittedParquetObject struct {
 	CreationDate    int64  `parquet:"name=creation_date, type=INT64, convertedtype=INT_64"`
 }
 
+// isValidEntry Returns False if entry is tombstone or if address is outside of repository namespace
+func isValidEntry(entry *UncommittedRecord) bool {
```

Comment:
That's a catalog-level function named `isValidEntry` which might be misused. Maybe
```suggestion
func shouldAddToUncommittedGC(entry *UncommittedRecord) bool {
```

Reply:
Removed function altogether